apps/headless/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>headless</title>
+    <script src='../../packages/util/dist/index.min.js'></script>
+    <script src='../../packages/common/dist/index.min.js'></script>
+    <script src='../../packages/api/dist/index.min.js'></script>
+    <script src='../../packages/chart/dist/index.min.js'></script>
+    <script src='../../packages/html/dist/index.min.js'></script>
+    <script src='../../packages/dgrid-shim/dist/index.min.js'></script>
+    <script src='../../packages/dgrid/dist/index.min.js'></script>
+    <script src='../../packages/layout/dist/index.min.js'></script>
+    <script src='../../packages/composite/dist/index.min.js'></script>
+    <script src='../../packages/phosphor-shim/dist/index.min.js'></script>
+    <script src='../../packages/phosphor/dist/index.min.js'></script>
+    <script src='../../packages/codemirror-shim/dist/index.min.js'></script>
+    <script src='../../packages/codemirror/dist/index.min.js'></script>
+    <script src='../../packages/graph/dist/index.min.js'></script>
+    <script src='../../packages/map/dist/index.min.js'></script>
+    <script src='../../packages/other/dist/index.min.js'></script>
+    <script src='../../packages/form/dist/index.min.js'></script>
+    <script src='../../packages/timeline/dist/index.min.js'></script>
+    <script src='../../packages/tree/dist/index.min.js'></script>
+</head>
```

Comment:
Would be better to use the "loader" then only the needed package(s) will get loaded and it will work while hosted on GitHub etc.